app/models/payment.rb

```diff
@@ -0,0 +1,23 @@
+class Payment < ApplicationRecord
+  ### CONSTANTS
+
+  STATUSES = {
+    paid: 0,
+    unpaid: 1
+  }
+
+  MODES = {
+    card: 0,
+    upi: 1
```

Comment:
Does stripe supports upi?
Have we integrated upi type of payment?

Reply:
No it doesn't accept upi. Removed it